content/post/agile-analytics-p2.md

```diff
@@ -0,0 +1,56 @@
+---
+author: Michael Kaminsky
+title: "Agile Analytics, Part 2: The Bad Stuff"
+publishDate: "2018-06-03"
+draft: true
+tags: 
+ - agile
+ - scrum
+ - process
+---
+
+Agile software engineering practices have become the standard work management tool for modern software development teams. Are these techniques applicable to analytics, or is the nature of research prohibitively distinct from the nature of engineering?
+
+
+I've already talked through what I [_like_]({{< relref "agile-analytics-p1.md" >}}) about the scrum development process for analytics. In this post, I'll talk through some of the problems and drawbacks.
+<!--more-->
+
+You can find [part 1 here]({{< relref "agile-analytics-p1.md" >}}).
+
+# Things that don't work so well:
+* The fortuitous finding
+* Exploratory data analysis
+* Product ownership / story-writing
+* Business-as-usual support
+
+I often say that analytics is a discipline that's half software engineering and half research. The parts of scrum that don't work so well for analytics are the parts that are more aligned with the "research" half of the analytics discipline.
+
+## The fortuitous finding
+
+Often, some of the highest value insights that an analytics team generate come from someone seeing something interesting -- but totally unrelated to their current assignment -- and then pursuing it.  This type of curiosity and tenacity is something we *love* to see in analysts we hire, but the scrum process, when it becomes a well-oiled box-checking process, can squash these types of endeavors.
```

Comment:
Is there not process for dealing with tangentially related work in agile? I can imagine a dev working on a feature enhancement and they find an unrelated bug in the code. That bug leads to another bug. Which calls for a major refactor and building new tests. This seems analogous to what you're suggesting an analyst might encounter, but agile doesn't appear to solve for. Could it be more a statement of how agile is implemented on a specific team/org vs. a failing of the philosophy of agile?